api/v1alpha1/client.go

```diff
@@ -14,14 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package client
+package v1alpha1
```

Comment:
it's not for this change maybe but AFAIK all other operators starts with v1beta1 so maybe You should change it in ovn-operator too

Reply:
Thanks slawek, if that's the case can do that in follow up.